fluent.runtime/docs/usage.rst

```diff
@@ -272,6 +272,25 @@ instances to indicate an error or missing data. Otherwise they should
 return unicode strings, or instances of a ``FluentType`` subclass as
 above.
 
+Attributes
+~~~~~~~~~~
+When rendering UI elements, it's handy to have a single translation that
+contains everything you need in one variable. For example, a Web
+Component confirm window with an OK button, a Cancel button, and a
+message.
```

Comment:
It might be better to use a signle HTML element as the example, as was done in the original comment on #209.